Atomic/atomic.py

```diff
@@ -379,8 +379,18 @@ def get_fq_name(self, image_info):
             if self.image in image_info['RepoTags']:
                 return self.image
 
+            possibles = []
+            for i in image_info['RepoTags']:
+                try:
+                    possibles.append(self.get_fq_image_name(i))
+                except RegistryInspectError:
+                    possibles.append(None)
```

Comment:
It appends None to the list, so later when I check if they all resolve to the same fq name, it will disqualify it.  I could theoretically raise the ValueError at this point to be quicker, but as of now, the final raise handles both cases.